src/components/mgt-tasks/mgt-tasks.ts

```diff
@@ -946,43 +1116,48 @@ export class MgtTasks extends MgtBaseComponent {
           >
             ${taskCheck}
           </span>
-          <span class="TaskTitle">
-            ${name}
-          </span>
-          ${taskDelete}
-        </div>
-        <div class="TaskDetails">
-          ${taskDresser} ${taskDrawer} ${taskPeople} ${taskDue}
+          <div class="TaskDetailsContainer">
+            ${taskDetails}
+          </div>
         </div>
+        ${taskOptions}
       </div>
     `;
   }
 
+  private handleClick(event, task: ITask) {
```

Comment:
this function needs a more descriptive name. How about `handleTaskClick`?

Reply:
renamed to `handleTaskClick` 00897b2